src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -92,8 +92,17 @@ const ResearchProfile = () => {
     <NotFound />
   ) : (
     <div>
-      <Box sx={{ bgcolor: 'person.main', py: '1.1rem', display: 'flex', alignItems: 'center', gap: '1rem' }}>
-        <Typography variant="h1" sx={{ ml: '2rem' }}>
+      <Box
+        sx={{
+          bgcolor: 'primary.main',
+          py: '1.1rem',
+          display: 'flex',
+          alignItems: 'center',
+          gap: '1rem',
+          borderLeft: 'solid 1rem',
+          borderColor: 'person.main',
```

Comment:
kunne vært helt eksplisitt på deg også, i tilfelle det dukker opp flere borders her en dag
```suggestion
          borderLeftColor: 'person.main',
```